src/Exception/InvalidConfigException.php

```diff
@@ -5,11 +5,16 @@
 use LogicException;
 use TomPHP\ExceptionConstructorTools;
 
+/**
+ * @api
```

Comment:
I am not entirely sure whether it is worth adding `@api` here - it seems like we are mixing blacklisting (using `@internal`) and whitelisting (using `@api`) at the same time.

Personally, I would just stick to black-listing, and using applying `@internal` only to classes which are not meant to be used externally. However, I'll leave it up to you, @tomphp. The downside of adding a lot of these annotations is that we need to remind ourselves to add them.

What do you think?